tests/inputFields.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { test, expect } from '@playwright/test';
+import exp from 'constants';
+
+test.beforeEach(async({page}) => {
+    await page.goto('https://petclinic.bondaracademy.com/welcome')
+})
+
+test.describe('Pet types', () => {
+    test.beforeEach(async({page}) => {
+        await page.getByRole('link', {name: "Pet Types"}).click() //1. Select Pet Types menu
+    })
+
+    test('Update pet type', async({page}) => {
+        const petTypesTableName = page.getByRole('heading', {name: "Pet Types"})
+        await expect(petTypesTableName).toHaveText('Pet Types')  //2. assertion of Pet Types text displayed
+
+        await page.getByRole('button', {name: "Edit"}).first().click()  //3. click on edit button for cat
+        await expect(page.getByRole('heading', {name: "Edit Pet Type"})).toBeVisible() //4. assertion of Edit Pet Type header
+
+        const editPetTypeName = page.locator('#name')
```

Comment:
The naming of the constant is a little bit misleading.
This is not a "pet type name". This is a locator for "pet type name input field".
Good naming is important for the code to be easier to read

Reply:
Updated constant to `editPetTypeNameInputField`